README.md

```diff
@@ -1 +1,72 @@
 # plextrac-manager-util
+
+This is the PlexTrac management CLI that can be used to perform the initial setup of a new PlexTrac installation as well as performing maintenance tasks on running instances.
+
+## Usage
+
+    Usage: plextrac COMMAND [FLAGS]
+
+    [+] Available commands:
```

Comment:
I pulled this list directly from the `help` output, but it looks like a few commands were/are missing (`logs`?).